manual/manual_mitres.rst

```diff
@@ -613,12 +617,12 @@ An extended explanation of these methods is available in the ``pytket`` `manual
 
    from qermit.frame_randomisation import FrameRandomisation
 
-   pfr_mitres = gen_Frame_Randomisation_MitRes(casablanca_real, 
+   pfr_mitres = gen_Frame_Randomisation_MitRes(lagos_, 
```

Comment:
Should there be something following this underscore?

Reply:
> Reference in new

updated!